app/models/statuses.py

```diff
@@ -5,9 +5,9 @@ def get_formatted(value: str):
     statuses = {    
     "NOT_STARTED": gettext("Not Started"),
     "IN_PROGRESS": gettext("In Progress"),
-    "COMPLETED": gettext("Assessment complete"),
-    "QA_COMPLETED": gettext("QA complete"),
-    "FLAGGED": gettext("Flagged"),
-    "STOPPED": gettext("Stopped"),
+    "COMPLETED": gettext("Completed"),
+    "SUBMITTED": gettext("Submitted"),
+    "NOT_SUBMITTED": gettext("Not Submitted"),
+    "READY_TO_SUBMIT": gettext("Ready to Submit"),
```

Comment:
Why are we introducing these statuses which I don't _think_ are used in the frontend?